content/en/_index.md

```diff
@@ -877,8 +877,14 @@ try {
 ### Linting {#linting}
 
 Our code snippets should adhere to a style used by the language communities.
-We prefer to enforce style using standard linters that are popular in
-the community.
+We follow the [PSR-1][psr-1] and [PSR-2][psr-2] community coding standards, with
```

Comment:
This section is cross-language. Could you add a  `content-tabpane` here similar to what we're using to wrap the text under **Language-specific practices** below?

Reply:
OK, I think I did it right